cpp/frameProcessor/include/FrameProcessorPlugin.h

```diff
@@ -90,45 +91,32 @@ class FrameProcessorPlugin : public IFrameCallback, public OdinData::IVersionedO
    */
   void add_metadata(OdinData::IpcMessage& message, const ParamMetadata& metadata) const
   {
-    std::string str = "metadata/";
-    static std::string& str_static = str;
-    static OdinData::IpcMessage&& message_static = std::move(message);
-
-    str += metadata.path;
-
+    std::string str = get_name() + metadata.path + '/';
```

Comment:
```suggestion
    std::string param_prefix = get_name() + metadata.path + '/';
```

The parameter in the set_param method that we are passing this full string to is called `param_name`, so this maps nicely.